phase2/addon/lib/timer.js

```diff
@@ -49,8 +49,6 @@ const _init = function(){
   if (!("silenceStart" in timerData))
     timerData.silenceStart = -1;
 
-  silenceLeft(); //to update the silence status;
```

Comment:
Silence status is not checked when timer is initializing. It's checked after 1 minute or when a recommendation is being delivered.